### PR TITLE
Separate IDEX L1 into L1a and L1b

### LINKED ISSUE #832: IDEX: Distinguish between L1a and L1b data products
The initial implementation of the IDEX L1 pipeline assumed that there was one L1 data product for IDEX. However, it has come to light that the IDEX team expects a separate L1a and L1b data product (with the L1b data product being basically the same thing as the L1a data product, but converted from units of DN to pC). The pipeline should be updated to make this distinction and produce separate products.

### PR DESCRIPTION
**Pull request by bourque**
The IDEX team informed us that they expect separate L1a and L1b data products (with the L1b data products being basically the same as the L1a data products, only converted from DN to pC). This PR makes various changes in order to distinguish between the two levels. Changes to create L1b products will be made in a separate PR.

I also took this PR as an opportunity to move the `idex_l0.py` and `idex_l1[a|b].py` modules into the same directory. In my opinion, keeping L0 and L1 code in separate subpackages is overkill.

### CODE REVIEW

**Comment by bourque** [comment]
Closes #832

File: docs/source/development-guide/style-guide/naming-conventions.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ The general filename convention is as follows::
   * CoDICE: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
   * GLOWS: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3a``, ``l3b``, ``l3c``, ``l3d``
   * HIT: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
-  * IDEX: ``l0``, ``l1``, ``l2``, ``l3``
+  * IDEX: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
```
Comment (greglucas) [inline docs/source/development-guide/style-guide/naming-conventions.rst:24]: Do we need to update the `sds-data-manager` repo with this as well for the dependency graph? If so, can you open an issue for that.

**Review by greglucas — APPROVED** [review]
I agree, this is a nicer organization.

File: docs/source/development-guide/style-guide/naming-conventions.rst (hunk 1 above)
Comment (bourque) [reply docs/source/development-guide/style-guide/naming-conventions.rst:24]: Ah yes, good call.

**Pull request merged**